components/Voting/sessionPanel.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Panel, Icon, Title, Badge, Details, Buttons } from './sessionPanel.styled'
+
+import SessionDetails from '../sessionDetails'
+
+export const SessionPanel: React.FC = ({
+  s,
+  i,
+  anonymousVoting,
+  isVotedFor,
+  votes,
+  maxVotes,
+  submitted,
+  isInShortlist,
+  expandAll,
+  toggleVote,
+  toggleShortlist,
+}) => {
+  return (
+    <Panel class="panel" key={s}>
+      <a href="#">
+        {isVotedFor && <Icon className="fa fa-check status" aria-label="Voted" role="status" title="Voted" />}
+        {isInShortlist && (
+          <Icon className="fa fa-list-ol status" aria-label="Shortlisted" role="status" title="Shortlisted" />
+        )}
+        <Title>{s.Title}</Title>
+        <ul>
+          {(s.Tags || []).map(tag => (
+            <Badge>{tag}</Badge>
+          ))}
+        </ul>
+        <div className="controls">
+          {!submitted && (
+            <Buttons>
+              <button
+                onClick={e => {
+                  toggleShortlist(s)
+                  e.stopPropagation()
+                  e.preventDefault()
```

Comment:
if you set the type of button, to button, then you wont need stopProp and preventDefault. It's trying to submit the page currently which is why they're needed.